src/pybind/mgr/dashboard_v2/tests/test_nodb.py

```diff
@@ -244,8 +244,9 @@ def get_all_objects(api_controller, query):
     def test_make_model_args(self):
         args = NodbModelTest.SimpleModel.make_model_args(dict(a=1, bad=3))
         self.assertEqual(args, dict(a=1))
-        args = NodbModelTest.SimpleModel.make_model_args(dict(a=1, bad=3), fields_force_none=['b'])
-        self.assertEqual(args, dict(a=1, b=None))
+        args = NodbModelTest.SimpleModel.make_model_args(dict(a=1, bad=3))
+        self.assertEqual(args, dict(a=1))
```

Comment:
It looks like 245-246 is the same as 247-248. If that's the case, can we remove one of both?